examples/axes_grid1/inset_locator_demo.py

```diff
@@ -44,7 +44,7 @@
 
 
 ###############################################################################
-# The arguments *bbox_to_anchor* and *bbox_transfrom* can be used for a more
+# The parameters *bbox_to_anchor* and *bbox_transfrom* can be used for a more
```

Comment:
There's another typo here:
```suggestion
# The parameters *bbox_to_anchor* and *bbox_transform* can be used for a more
```

Reply:
I completely missed that, thanks for the correction!